### PR TITLE
[Keyboard] Rename dnworks/sbl to dnworks/60percent

### PR DESCRIPTION
**Pull request by itsme-zeix**
## Description
Renamed dnworks/sbl to dnworks/60percent due to different 60% PCBs that are electronically the same. Included the rename in data/mappings/keyboard_aliases.hjson.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Review by duncsuth-1adv — APPROVED** [review]

**Review by dunk2k — APPROVED** [review]

File: data/mappings/keyboard_aliases.hjson [hunk 1]
```diff
@@ -1531,5 +1531,8 @@
     },
     "dnworks/9973": {
         "target": "dnworks/tkl87"
-    }
+    },
+    "dnworks/sbl": {
+        "target": "dnworks/60percent"
+    },
```
Comment (fauxpark) [inline data/mappings/keyboard_aliases.hjson:1537]:
```suggestion
    }
```

File: keyboards/dnworks/60percent/readme.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 
 See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
 
 ## Bootloader
 
 Enter the bootloader in 3 ways:
 
-* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
-* **Physical reset button**: Short the 'USB_BOOT' button and plug in keyboard or briefly short the `RESET` and `GND` pads on the SWD header twice
+* **Bootmagic reset**: Hold down escape and plug in the keyboard
```
Comment (fauxpark) [inline keyboards/dnworks/60percent/readme.md:25]:
```suggestion
* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by tzarc** [comment]
As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.

As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.

**Comment by itsme-zeix** [comment]
> As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.
> 
> As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.

Removed `./keymaps/via`.

**Comment by drashna** [comment]
There is a merge conflict here

**Pull request closed without merging**